src/web/lib/components/AppFooter/index.js

```diff
@@ -9,74 +9,78 @@ import iconTwitter from "./twitter-logo.svg";
 
 import "./index.scss";
 
-export const AppFooter = () => (
-  <footer className="app-footer">
-    <nav className="app-footer__legal">
-      <a
-        className="app-footer__legal-link"
-        onClick={() => Metrics.linkClick("mozilla-logo")}
-        href="https://www.mozilla.org"
-        aria-label="Mozilla logo"
-      >
-        <ReactSVG path={iconMoz} className="app-footer__legal-logo" />
-      </a>
-      <a
-        className="app-footer__legal-link"
-        onClick={() => Metrics.linkClick("legal")}
-        href="https://www.mozilla.org/about/legal"
-      >
-        Legal
-      </a>
-      <a
-        className="app-footer__legal-link"
-        onClick={() => Metrics.linkClick("about")}
-        href="https://testpilot.firefox.com/about"
-      >
-        About Test Pilot
-      </a>
-      <a
-        className="app-footer__legal-link"
-        onClick={() => Metrics.linkClick("privacy")}
-        href="/legal"
-      >
-        Privacy
-      </a>
-      <a
-        className="app-footer__legal-link"
-        onClick={() => Metrics.linkClick("terms")}
-        href="/legal"
-      >
-        Terms
-      </a>
-      <a
-        className="app-footer__legal-link"
-        onClick={() => Metrics.linkClick("cookies")}
-        href="https://www.mozilla.org/privacy/websites/#cookies"
-      >
-        Cookies
-      </a>
-    </nav>
-    <nav className="app-footer__social">
-      <a
-        className="app-footer__social-link"
-        onClick={() => Metrics.linkClick("twitter")}
-        href="https://twitter.com/FxTestPilot"
-        aria-label="@FxTestPilot Twitter"
-        title="Twitter"
-      >
-        <ReactSVG path={iconTwitter} className="app-footer__social-logo" />
-      </a>
-      <a
-        className="app-footer__social-link"
-        onClick={() => Metrics.linkClick("github")}
-        href="https://github.com/mozilla/FirefoxColor/"
-        aria-label="Firefox Color GitHub"
-        title="GitHub"
-      >
-        <ReactSVG path={iconGH} className="app-footer__social-logo" />
-      </a>
-    </nav>
-  </footer>
-);
+export const AppFooter = ({ setDisplayLegalModal }) => {
+  const toggleModal = name => {
```

Comment:
Added modal controls to the the footer so we can trigger appropriate actions on terms/privacy clicks